### ISSUE TITLE
Route rating by users

### ISSUE THREAD

**Issue by Galven** [zugspitze/Hiking-Buddies#20]
We already talked about the way to rate a route's difficulty. We sort of agreed on the following system.

There are three components:
- Physical difficulty
- Technical difficulty
- Exposition

For each routes, people will have the possibility of rating it according to those components.

You suggested for technical difficulty to keep being calculated automatically according to SAC Scale, but we'll have a problem for countries without information about that scale.

Then we'd have two choices:
1 - **Make it 4 components for all countries**. People would be able to rate the technical difficulty as well, in addition to which we'd have the automatic calculation according to SAC scale when available.
2 - **Keep it 3 components**. If SAC information available, people wouldn't be able to rate the technical difficulty (that would be automatic), but if not available, people would rate also the technical difficulty themselves.

I'd vote for the second one.

Example

![1](https://user-images.githubusercontent.com/40529268/42057669-a254b0d6-7b0d-11e8-876e-e584da5aa15e.JPG)

![2](https://user-images.githubusercontent.com/40529268/42057673-a436326c-7b0d-11e8-9cb7-f84041bafe98.JPG)

**Comment by zugspitze** [comment]
Yes, we agreed on such system. But unfortunately I didn't find time to make it. Thanks for making notes here.
